wezterm-gui/src/overlay/confirm.rs

```diff
@@ -0,0 +1,216 @@
+use crate::scripting::guiwin::GuiWin;
+use config::keyassignment::{Confirmation, KeyAssignment};
+use mux::termwiztermtab::TermWizTerminal;
+use mux_lua::MuxPane;
+use std::rc::Rc;
+use termwiz::cell::AttributeChange;
+use termwiz::color::ColorAttribute;
+use termwiz::input::{InputEvent, KeyCode, KeyEvent, MouseButtons, MouseEvent};
+use termwiz::surface::{Change, CursorVisibility, Position};
+use termwiz::terminal::Terminal;
+
+pub fn run_confirmation(message: &str, term: &mut TermWizTerminal) -> anyhow::Result<bool> {
+    run_confirmation_impl(message, term)
+}
+
+fn run_confirmation_impl(message: &str, term: &mut TermWizTerminal) -> anyhow::Result<bool> {
+    term.set_raw_mode()?;
+
+    let size = term.get_screen_size()?;
+
+    /*
+    use crate::termwindow::ICON_DATA;
+    use image::GenericImageView;
+    use std::sync::Arc;
+    use termwiz::surface::change::ImageData;
+    use termwiz::surface::TextureCoordinate;
+
+        let image_data = Arc::new(ImageData::with_raw_data(ICON_DATA.to_vec()));
+        let decoded_image = image::load_from_memory(ICON_DATA)?;
+        let logo_width_cells = 3usize;
+        let logo_width = logo_width_cells * size.xpixel;
+        let logo_scale = logo_width as f32 / decoded_image.width() as f32;
+        let logo_height = decoded_image.height() as f32 * logo_scale;
+        let logo_height_cells = (logo_height / size.ypixel as f32).ceil() as usize;
+        */
```

Comment:
I know this is moving from existing code, but I think it's fine to just delete this dead commented code; it won't serve any purpose over here